website/app/main.js

```diff
@@ -18,7 +18,7 @@ window.MacMessenger = {
   showSettings: function() {
     this.openGearMenu();
     document.querySelector(
-      'div.uiLayer.uiContextualLayerPositioner#js_1 > div.uiContextualLayer a'
+      'div.uiLayer.uiContextualLayerPositioner#js_2 > div.uiContextualLayer a'
```

Comment:
Change the `[title*="Settings"]` to the ID then? `#js_1u`. Anyway, the rest should make sense.